{
    "checksum": "9f02feba6e7515694b566e4974be25f13b74caa8c0c0fa03df4a711582e7ee0e",
    "file": "wrong-file-sha1-hash-in-gz-archive.conf",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 6,
    "package": "backupchecker",
    "path": "backupchecker/1.7-3/functional-tests/wrong-file-sha1-hash-in-gz-archive/wrong-file-sha1-hash-in-gz-archive.conf",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/backupchecker/1.7-3/",
        "metric": {
            "size": 6452
        },
        "pts_link": "https://tracker.debian.org/pkg/backupchecker",
        "sloc": [
            [
                "python",
                8568
            ],
            [
                "makefile",
                161
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/packages/backupchecker",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/backupchecker/1.7-3/functional-tests/wrong-file-sha1-hash-in-gz-archive/wrong-file-sha1-hash-in-gz-archive.conf",
    "stat": {
        "perms": "rw-r--r--",
        "size": 156,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.7-3"
}